mobile/src/main/res/raw/truckmuncher.sql

```diff
@@ -9,7 +9,8 @@ CREATE TABLE `truck_properties` (
 	`image_url`	                TEXT,
 	`keywords`	                TEXT,
 	`color_primary`	            TEXT,
-	`color_secondary`	        TEXT
+	`color_secondary`	        TEXT,
+	`owned_by_current_user`     INTEGER DEFAULT 0
```

Comment:
I feel like this would be more appropriate on the state rather than a property. Reason being, if we expose the data to other apps, it would be weird to have this change just because a user logged out in our app. Also, think it should be added to the `Truck` view?